packages/experimental/Button/src/CompoundButton/CompoundButton.tsx

```diff
@@ -29,10 +29,24 @@ const CompoundButtonComposed = compose<CompoundButtonType>({
 
     // now return the handler for finishing render
     return (final: CompoundButtonPropsWithInnerRef, ...children: React.ReactNode[]) => {
-      const { icon, secondaryContent, iconPosition, ...mergedProps } = mergeProps(button.props, final);
+      const { icon, secondaryContent, iconPosition, accessibilityLabel, ...mergedProps } = mergeProps(button.props, final);
+      let childText = '';
+      if (accessibilityLabel === undefined) {
+        React.Children.forEach(children, (child) => {
+          if (typeof child === 'string') {
+            childText = child; // We only automatically support the one child string.
+          }
+        });
+      }
+
+      if (secondaryContent) {
```

Comment:
This can be moved in the above if statement (after line 39). If accessibilityLabel !== undefined, this logic doesn't matter, and thus should be moved (small perf reasons :) )

Reply:
Thanks for catching, done.